netctl/commands.go

```diff
@@ -168,27 +174,37 @@ var Commands = []cli.Command{
 				Name:      "ls",
 				Aliases:   []string{"list"},
 				Usage:     "List rules for a given tenant/policy",
-				ArgsUsage: "[policy]",
-				Flags:     []cli.Flag{tenantFlag, allFlag, jsonFlag},
-				Action:    listRules,
+				ArgsUsage: "",
+				Flags: []cli.Flag{tenantFlag, allFlag, jsonFlag, quietFlag,
+					cli.StringFlag{
+						Name:  "policy, p",
+						Usage: "Policy - REQUIRED",
+					},
+				},
+				Action: listRules,
 			},
 			{
 				Name:      "rm",
 				Aliases:   []string{"delete"},
 				Usage:     "Delete an existing rule.",
-				ArgsUsage: "[policy] [rule id]",
-				Flags:     []cli.Flag{tenantFlag},
-				Action:    deleteRule,
+				ArgsUsage: "[rule id]",
```

Comment:
@vijayvikrant on a side note, any changes to netctl CLI will require corresponding documentation changes too